posthog/queries/trends/trend_event_query.py

```diff
@@ -39,6 +39,12 @@ def get_query(self) -> Tuple[str, Dict[str, Any]]:
                 ]
             )
             + (f", {self.DISTINCT_ID_TABLE_ALIAS}.person_id as person_id" if self._should_join_distinct_ids else "")
+            + (
+                f", {self.SESSION_TABLE_ALIAS}.session_duration as session_duration"
+                if self._should_join_sessions
+                else ""
+            )
+            + (f", {self.SESSION_TABLE_ALIAS}.$session_id as $session_id" if self._should_join_sessions else "")
```

Comment:
Might be worth only pulling these columns when we need them. For example, we don't need them in the generic "unique sessions" query. Could be an over optimization too.

Reply:
Yep yep. Makes total sense

Reply:
Bleh, doesn't seem worth it right now :P - it's not a JSON blob we're querying here, but a number, and it's limited to aggregating by sessions, so seems fine to me 🤷 .


I could go the `column_optimizer` route, but really want one more property other than `$session_duration` before I try generalising and querying from there.